content/docs/command-reference/exp/run.md

```diff
@@ -84,13 +84,23 @@ steps as `make_checkpoint()` — please refer to its reference for details.
 You can now use `dvc exp run` to begin the experiment. If the process gets
 interrupted (e.g. with `[Ctrl] C` or by an error), all the checkpoints so far
 will be preserved. When a run finishes normally, a final checkpoint will be
-added (if needed) to wrap up the experiment.
+added (if needed) to wrap up the experiment. Following uses of `dvc exp run`
+will continue from this point (using the latest cached versions of all outputs).
 
-Following uses of `dvc exp run` will continue from this point (using the latest
-cached versions of all outputs). You can add a `--rev` to continue from a
-previous checkpoint instead (list them with `dvc exp show`). Or use `--reset` to
-start over (discards previous checkpoints and deletes `checkpoint` outputs, like
-the first `dvc exp run`) — useful for re-training ML models, for example.
+To continue from a previous checkpoint (list them with `dvc exp show`), when
+running experiments in your workspace, you must first use `dvc exp apply` to
+apply the checkpoint from which you want to resume before using `dvc exp run`.
+When running experiments outside of your workspace via `--queue/--run-all` or
+`--temp`, you can use `--rev` to directly specify the checkpoint from which you
+want to resume.
+
+⚠️ Note that `--rev` is disallowed when running in your workspace, since using
+`--rev` in this way could potentially lead to discarding and overwriting changes
+in your workspace.
+
+Alternatively, use `--reset` to start over (discards previous checkpoints and
+deletes `checkpoint` outputs, like the first `dvc exp run`) — useful for
+re-training ML models, for example.
```

Comment:
```suggestion
Alternatively, use `--reset` to start over (discards previous checkpoints and
their outputs, like things as they were before the first `dvc exp run`). This is
useful for re-training ML models, for example.
```

Reply:
Def. better here as well, but

> like things as they were before

We literally meant "like the first `dvc exp run` (does)". Meaning that the first `dvc exp run` deletes checkpoint outputs. The above suggestion would say "discards things as they were before". Maybe let's try to simplify the sentence, maybe remove split into 2 without parentheses? Not sure

Reply:
I meant to write "leaving," not "like." Better now?

Reply:
Yes, thanks. Let me commit this but actually I realized we decided at some point not to document the "first `exp run` edge case so we could just remove that parenthesis 😆. I'll make a separate suggestion... ⌛